Documentation/_static/helm-reference.css

```diff
@@ -0,0 +1,12 @@
+#helm-reference .wy-table-responsive table td {
+    white-space: normal!important;
```

Comment:
Are the `important` tags needed?

Reply:
No, they stem from my initial testing — thanks for spotting!